Xwt.XamMac/Xwt.Mac/CanvasBackend.cs

```diff
@@ -52,6 +52,8 @@ protected override void OnSizeToFit ()
 			Widget.SetFrameSize (new CGSize ((nfloat)s.Width, (nfloat)s.Height)); 
 		}
 
+		public override Xwt.Drawing.Color BackgroundColor { get; set; }
```

Comment:
```suggestion
		// Skip Layer initialization in base to avoid rendering issues. CanvasView.DrawRect will draw the background.
		public override Xwt.Drawing.Color BackgroundColor { get; set; }
```